internal/display/user_blocks.go

```diff
@@ -0,0 +1,56 @@
+package display
+
+import (
+	"gopkg.in/auth0.v5/management"
+)
+
+type userBlockView struct {
+	Identifier string
+	IP         string
+}
+
+func (v *userBlockView) AsTableHeader() []string {
+	return []string{"Identifier", "IP"}
+}
+
+func (v *userBlockView) AsTableRow() []string {
+	return []string{v.Identifier, v.IP}
+}
+
+func (v *userBlockView) KeyValues() [][]string {
+	return [][]string{
+		[]string{"Identifier", v.Identifier},
+		[]string{"IP", v.IP},
+	}
+}
+
+func (r *Renderer) UserBlocksList(userBlocks []*management.UserBlock) {
+	resource := "userBlocks"
+
+	r.Heading(resource)
+
+	if len(userBlocks) == 0 {
+		r.EmptyState(resource)
+		r.Infof("No blocks for user.")
```

Comment:
`r.EmptyState(resource)` and `r.Infof("No blocks for user.")` do the same thing, please remove the last one.